common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/RetryingBlockFetcher.java

```diff
@@ -178,7 +202,11 @@ private synchronized boolean shouldRetry(Throwable e) {
     boolean isIOException = e instanceof IOException
```

Comment:
what is the exception type when the executor is dead during shuffle fetch?

Reply:
It's IOException, just like bellow
```
2019-04-26 12:18:50,291 [26151] - WARN  [shuffle-client-4-1:TransportChannelHandler@78] - Exception in connection from hostname/ipAddress:port
java.io.IOException: Connection reset by peer
```